src/app/kbv/steps.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
   "/load-question": {
     controller: loadQuestionController,
     skip: true,
-    next: loadQuestionController.prototype.next,
+    next: [
+      { fn: loadQuestionController.prototype.hasQuestion, next: "question" },
```

Comment:
Why `loadQuestionController.prototype.hasQuestion` and not just `loadQuestionController.hasQuestion`? The `hasQuestion` is defined on the top-level class, isn't it? Or am I not understanding something about the JS inheritance model?

Reply:
I'm following the example in the documentation - https://github.com/HMPO/hmpo-form-wizard?tab=readme-ov-file#next-steps